src/main/java/gift/controller/GlobalExceptionHandler.java

```diff
@@ -1,18 +1,32 @@
 package gift.controller;
 
 import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.validation.FieldError;
 import org.springframework.web.bind.MethodArgumentNotValidException;
-import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
 
-@ControllerAdvice
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestControllerAdvice
 public class GlobalExceptionHandler {
 
     @ExceptionHandler(MethodArgumentNotValidException.class)
-    @ResponseStatus(HttpStatus.BAD_REQUEST)
-    public String handleValidationExceptions() {
-        return "version-SSR/product-error";
+    public ProblemDetail handleValidationExceptions(MethodArgumentNotValidException e) {
+        ProblemDetail problemDetail = ProblemDetail.forStatus(HttpStatus.BAD_REQUEST);
+        problemDetail.setTitle("상품 유효성 검사 실패");
+        problemDetail.setProperty("rejectedValue",e.getFieldError().getRejectedValue());
```

Comment:
코드를 작성한 후 정렬하는 습관을 들여주세요!  
전체 코드에 대해 정렬을 수행해주세요.

```suggestion
        problemDetail.setProperty("rejectedValue", e.getFieldError().getRejectedValue());
```

Reply:
네 유의하겠습니다!